content/en/docs/kitex/Tutorials/advanced-feature/multi_service.md

```diff
@@ -22,8 +22,11 @@ Currently, the feature is available for:
 If you are client-side users and using the multi-service feature, please follow the instructions below:
 
 1. Upgrade the client to the Kitex version >= v0.9.0
+
+Note: 2 and 3 below are only for Kitex thrift & protobuf (non-streaming) users.
```

Comment:
This format is a bit weird.

I suggest making it:

1. xxx
2. For Kitex thrift & protobuf (non-streaming) APIs:

  1. TTHeader
  2. ClientTTHeaderHandler

Reply:
Since these info is repeated several times, how about make it an independent section so the rest occurencies can refer to it?